ocw_data_parser/ocw_data_parser.py

```diff
@@ -377,6 +386,12 @@ def upload_all_media_to_s3(self, chunk_size=1000000, upload_master_json=False):
                                    aws_secret_access_key=self.s3_bucket_secret_access_key
                                    ).Bucket(self.s3_bucket_name)
         # Upload static files first
+        for p in self.compose_pages():
+            if safe_get(p, "text"):
+                file_name = safe_get(p, "uid") + "_" + safe_get(p, "short_url") + ".html"
+                html = "<html><head></head><body>" + safe_get(p, "text") + "</body></html>"
```

Comment:
Minor nit: this is pretty similar to lines 312-313 above, maybe a new function could be called in each spot, something like:
```python
def htmlify(page):
            safe_text = safe_get(page, "text")
            if safe_text:
                file_name = safe_get(page, "uid") + "_" + safe_get(page, "short_url") + ".html"
                html = "<html><head></head><body>" + safe_text + "</body></html>"
                return file_name, html
            return None, None
.......
filename, html = htmlify(p)
if filename and html:
      do_stuff
```

But it works fine as is, so up to you

Reply:
I like this, let's do it this way.